scraper/scraper/spiders/clients_spider.py

```diff
@@ -54,11 +55,13 @@ def after_login(self, response):
             r = requests.get(self.get_client_booking_url,
                              params={'id': user}, cookies=jar)
             data = r.json()
```

Comment:
Add 2 checks here to only process customers that:
1. have been active within the last 2 years
2. have either a valid email or phone number only